orc/src/main/java/org/apache/iceberg/orc/OrcFileAppender.java

```diff
@@ -126,12 +100,14 @@ public long length() {
   @Override
   public List<Long> splitOffsets() {
     Preconditions.checkState(isClosed, "File is not yet closed");
+    String fileLoc = file.location();
     Reader reader;
     try {
-      reader = OrcFile.createReader(path, new OrcFile.ReaderOptions(conf));
+      reader = OrcFile.createReader(new Path(fileLoc), new OrcFile.ReaderOptions(conf));
```

Comment:
This seems expensive. Up to you whether to implement it or just let split calculation happen without split locations.

Reply:
Seems like the stripe offsets are used as splits here. What do you think @rdsr since this was introduced in https://github.com/apache/incubator-iceberg/pull/192 ? - I'm inclined to create an issue for this an address in other PR?

Reply:
Ideally, the stripe offsets would come from the writer, not reading the file after it is written.